Draft v1.0/Specification.md

```diff
@@ -94,7 +94,7 @@ The following table describes the requirements for data quality when preserving
 | --- | --- | --- |
 | ODS_1 | Password protection | The file **MUST NOT** have read or write password protection. |
 | ODS_2 | Standard compliance | The file **MUST** comply with the standard “OASIS Open Document Format for Office Applications (OpenDocument) v1.3”. |
-| ODS_3 | Extension | The file **MUST** have extension “.fods”, “.ods” or “.ots”. |
+| ODS_3 | Extension & MIME type | The file **MUST** have one of the following matching extension and MIME type pairs:<ul><li>Extension: ".fods", MIME type: "application/vnd.oasis.opendocument.spreadsheet"</li><li>Extension: ".ods", MIME type: "application/vnd.oasis.opendocument.spreadsheet"</li><li>Extension: ".ots", MIME type: "application/vnd.oasis.opendocument.spreadsheet-template"</li></ul> |
```

Comment:
There is already a consistency table in the ODF specification, might be better to refer to that and simply list the MIME type: https://docs.oasis-open.org/office/OpenDocument/v1.3/os/part3-schema/OpenDocument-v1.3-os-part3-schema.html#__RefHeading__1420388_253892949

Reply:
I would keep the pairs in. Otherwise, we can just change the existing one to refer to the MIME type instead. The result will be the same.